rules_bot.py

```diff
@@ -138,55 +144,50 @@ def main() -> None:
     )
 
     # Inline Queries
-    inlinequeries.register(dispatcher)
+    dispatcher.add_handler(InlineQueryHandler(inlinequeries.inline_query))
+
+    # Captcha for userbots
+    dispatcher.add_handler(CommandHandler("say_potato", say_potato_command))
```

Comment:
this is an admin only command right? Ik there's a scope on it, but doesn't stop users from using it. I didn't see any check for admin in its callback, is it implemented?

Reply:
good catch!